subsys/bluetooth/audio/tbs.c

```diff
@@ -817,20 +782,13 @@ static ssize_t read_incoming_uri(struct bt_conn *conn,
 					    void *buf, uint16_t len,
 					    uint16_t offset)
 {
+	struct service_inst *inst = BT_AUDIO_CHRC_USER_DATA(attr);
 	const struct bt_tbs_in_uri *inc_call_target;
 	size_t val_len;
 
-	if (IS_GTBS_CHRC(attr)) {
-		inc_call_target = &gtbs_inst.incoming_uri;
-		LOG_DBG("GTBS: call index 0x%02X, URI %s", inc_call_target->call_index,
-			inc_call_target->uri);
-	} else {
-		const struct tbs_service_inst *inst = BT_AUDIO_CHRC_USER_DATA(attr);
-
-		inc_call_target = &inst->incoming_uri;
-		LOG_DBG("Index %u: call index 0x%02X, URI %s", inst->index,
-			inc_call_target->call_index, inc_call_target->uri);
-	}
+	inc_call_target = &inst->incoming_uri;
+	LOG_DBG("Index %u: call index 0x%02X, URI %s", inst_index(inst),
```

Comment:
minor observation: there seem to be a mix of % X, x and u for print of call index and other values in the file